drafts/2016-01-06.md

```diff
@@ -0,0 +1,91 @@
+Welcome to the first edition of IPFS Weekly!  [IPFS](https://ipfs.io/) is a new hypermedia distribution protocol, addressed by content and identities, aiming to make the web faster, safer, and more open.  In these posts, we will try to highlight some of development that happened in the past week.  For anyone looking to get involved, follow the embedded hyperlinks, search the wealth of information on [github](https://github.com/ipfs) or join us on [IRC](http://webchat.freenode.net/?channels=ipfs) (#ipfs on the Freenode network).
+
+Since this is our first time launching the Weekly, we've included several past weeks. This is partially because we've been refining our process, and wanted to make the first weekly a great one. In the future, they will be released weekly. For now, here is a summary of what has happened over at IPFS for the past four weeks. If you have any feedback about this process in general, let us know [here](https://github.com/ipfs/weekly/issues/7). Thanks!
+
+## December 21
+
+Here are some of the highlights for the [December 21](https://github.com/ipfs/pm/issues/76) Sprint:
+
+Not too much happened during this sprint, because it was the holidays. [@jbenet](//github.com/jbenet) was off visiting Middle Earth (New Zealand), and [@whyrusleeping](//github.com/whyrusleeping), [@diasdavid](//github.com/diasdavid), [@lgierth](//github.com/lgierth), and more of the team were over at 32c3 in Hamburg.
```

Comment:
We should have some notes about 32c3, in particular:

- @whyrusleeping gave a lightning talk (we may be able to include the picture below)
- @whyrusleeping @diasdavid and @Dignifiedquire led an install party and live demo (picture below)
- @MrChrisJ led a discussion about his [Full Node project](https://news.bitcoin.com/developer-chris-ellis-builds-full-bitcoin-nodes-12-hours/), which includes IPFS.
- met lots of our users, other groups (including Tahoe-LAFS, tor, etc).

![](https://ipfs.io/ipfs/QmWYaX56pTskFL2UMV2x8ZZAB8xyntPce5bFmY8ugHw47j/32c3.ipfs.2.jpg)
![](https://ipfs.io/ipfs/QmWYaX56pTskFL2UMV2x8ZZAB8xyntPce5bFmY8ugHw47j/32c3.ipfs.1.jpg) 
(we should only include the second, if whoever took the picture asked if it was ok to share the picture widely)